lib/evaluate/intrinsics.cc

```diff
@@ -1651,6 +1653,51 @@ std::optional<SpecificCall> IntrinsicProcTable::Implementation::Probe(
       } else {
         specificBuffer.clear();
       }
+      // If there was no exact match with a specific, try to match the related
```

Comment:
This line length exceeds 80 characters.

Reply:
This ```Probe()``` function is very long, which reduces its readability.

Reply:
I agree it is not the most readable function after my change. I have pushed a new commit that does not make it much shorter but makes all the logic and paths much more easy to follow (IMHO). It mainly hides the message buffer problem a bit.
@psteinfeld, is it more readable ?
@klausler,  I have removed the usage of a local buffer for the `NULL` handling, it seemed not useful to me since it was always copied into the actual context anyway. Do you see a reason that would justify keeping it ?

Reply:
Thanks, Jean.  This is a gigantic improvement.